src/main/java/core/basesyntax/operation/BuyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.Storage;
+import core.basesyntax.model.Transaction;
+
+public class BuyOperation implements Operational {
+
+    @Override
+    public void operation(Transaction transaction, Storage fruitStorage) {
```

Comment:
let's not pass Storage to method argument. Rather pass it as a constructor parameter

Reply:
done

Reply:
done